File: programme/perimeter-of-square/PerimeterOfSquare.java
// Write a programme to find perimeter of square
import java.util.Scanner;

class PerimeterOfSquare {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);

        System.out.print("side : ");
        int s = Integer.parseInt(sc.nextLine());
        s = 4 * s;
        System.out.println("\nPerimeter : " + s);
        sc.close();
    }
}

File: programme/perimeter-of-square/perimeter-of-square.js
// Write a programme to find perimeter of square

"use strict";

// perform following command to run programme
// node perimeter-of-square.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question('side : ', n => {
	const p = parseInt(n) * 4;
	if (isNaN(p)) {
		console.error('Some error occured');
	} else {
		console.log(`perimeter : ${p}`);
	}
	rl.close();
});
